client/components/inline-support-link/index.jsx

```diff
@@ -116,7 +117,10 @@ class InlineSupportLink extends Component {
 			<LinkComponent
 				className={ classnames( 'inline-support-link', className ) }
 				href={ url }
-				onClick={ ( event ) => openDialog( event, supportPostId, url ) }
```

Comment:
This function used to return `openDialog`, but no longer does - is that a concern, and was it intended?

Reply:
I am not sure. In my testing, everything worked as expected for me. 

I update the code to return the same as before thought.